packages/cardpay-sdk/sdk/prepaid-card/base.ts

```diff
@@ -64,22 +64,36 @@ export default class PrepaidCard {
     return !hasBeenUsed && owner === issuer && owner !== ZERO_ADDRESS;
   }
 
+  async payMerchant(txnHash: string): Promise<TransactionReceipt>;
   async payMerchant(
     merchantSafe: string,
     prepaidCardAddress: string,
     spendAmount: number,
     txnOptions?: TransactionOptions,
     contractOptions?: ContractOptions
+  ): Promise<TransactionReceipt>;
+  async payMerchant(
+    merchantSafeOrTxnHash: string,
+    prepaidCardAddress?: string,
+    spendAmount?: number,
+    txnOptions?: TransactionOptions,
+    contractOptions?: ContractOptions
   ): Promise<TransactionReceipt> {
-    if (spendAmount < 50) {
+    if (!prepaidCardAddress) {
```

Comment:
maybe a better test is that `merchantSafeOrTxnHash` actually looks like an address, that way you are asserting on the value you care about rather than making an assertion on a side effect. All addresses have 40 characters plus "0x" (so 42 characters in total). All txn hashes have 64 characters plus "0x" (so 66 chars in total).